pkgs/applications/graphics/ahoviewer/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, pkgs, fetchurl, fetchFromGitHub, pkgconfig, libconfig, gtkmm, glibmm, libxml2, gstreamer, libsecret, curl, unrar, libzip, autoconf, automake }:
+with stdenv.lib;
+stdenv.mkDerivation rec {
+  name = "ahoviewer-1.4.6";
+  rev = "414cb91d66d96fab4b48593a7ef4d9ad461306aa";
+  src = fetchFromGitHub {
+    owner = "ahodesuka";
+    repo = "ahoviewer";
+    inherit rev;
```

Comment:
You aren't using `rev` anywhere outside this set. Therefore you can directly define it here. There is no need to define it directly in the package derivation if you don't need it anywhere else. But once again nothing crucial.